Dao/src/main/java/com/epam/ems/dto/Certificate.java

```diff
@@ -0,0 +1,84 @@
+package com.epam.ems.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Certificate {
+    private int id;
+    private String name;
+    private String description;
+    private int price;
+    private int duration;
+    private String createDate;
+    private String lastUpdateDate;
+    private List<Tag> tags;
+
+    public Certificate(String name, String description, int price, int duration, String createDate, String lastUpdateDate, List<Tag> tags) {
+        this.name = name;
+        this.description = description;
+        this.price = price;
+        this.duration = duration;
+        this.createDate = createDate;
+        this.lastUpdateDate = lastUpdateDate;
+        this.tags = new ArrayList<>(tags);
+    }
+
+    public Certificate(int id, String name, String description, int price, int duration, String createDate, String lastUpdateDate, List<Tag> tags) {
+        this.id = id;
+        this.name = name;
+        this.description = description;
+        this.price = price;
+        this.duration = duration;
+        this.createDate = createDate;
+        this.lastUpdateDate = lastUpdateDate;
+        this.tags = new ArrayList<>(tags);
+    }
```

Comment:
These are almost similar constructors. Use this() conctructor to remove duplications.